client.go

```diff
@@ -140,20 +140,25 @@ type Client interface {
 
 	// Exec command, support mul-commands in one transaction.
 	Execute(cmds ...*OvnCommand) error
+
+	// Close connection to OVN
+	Close() error
 }
 
 type ovndb struct {
-	client     *libovsdb.OvsdbClient
-	cache      map[string]map[string]libovsdb.Row
-	cachemutex sync.RWMutex
-	tranmutex  sync.Mutex
-	callback   OVNSignal
+	client       *libovsdb.OvsdbClient
+	cache        map[string]map[string]libovsdb.Row
+	cachemutex   sync.RWMutex
+	tranmutex    sync.Mutex
+	callback     OVNSignal
+	disconnected OVNDisconnectedCallback
```

Comment:
may be rename this like disconnectCB and callback to signalCB ?

Reply:
Indeed. Done